test/extensions/quic_listeners/quiche/platform/quic_system_event_loop_impl.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+inline void QuicRunSystemEventLoopIterationImpl() {
+  // No-op.
+}
+
+class QuicSystemEventLoopImpl {
+public:
+  // Only used by quic_client_bin.cc which is not required in Envoy.
```

Comment:
lazy question - if it's only used by a file not required by Envoy, why can't we just skip implementing it?

Reply:
QuicRunSystemEventLoopIterationImpl is used by QuicClientEpollNetworkHelper which is needed to run quic e2e tests. Though QuicSystemEventLoopImpl is not used, it is bundled in same API file as QuicRunSystemEventLoopIterationImpl, so it need to be defined.

Reply:
Ah, that's a bummer - thanks for the explanation.  

LGTM modulo clang tidy warning, and master merge.

Reply:
done